crates/bevy_app/Cargo.toml

```diff
@@ -13,7 +13,6 @@ trace = []
 bevy_debug_stepping = []
 default = ["bevy_reflect"]
 bevy_reflect = ["dep:bevy_reflect", "bevy_ecs/bevy_reflect"]
-serialize = ["bevy_ecs/serde"]
```

Comment:
Does `bevy_app` really not have any other serialization functionality?

Reply:
Ctrl+F showed 0 hits!